keyboards/keebio/levinson/keymaps/Rossman360_linux/keymap.c

```diff
@@ -0,0 +1,191 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define EE_HANDS
+
+#define _QWERTY 0
+#define _DEL 1
+#define _FN1 2
+#define _FN2 3
+#define _NUM 4
```

Comment:
You should use an `enum` here:

```c
enum layer_names {
    _QWERTY,
    _DEL,
    _FN1,
    _FN2,
    _NUM,
};
```

Reply:
reverted this file to the way it is on master branch. long story of epic git struggle.